sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -231,7 +239,7 @@ class GpuWindowExecMeta(windowExec: WindowExec,
   override def getResultColumnsOnly: Boolean = resultColumnsOnly
 }
 
-object GpuWindowExec extends Arm {
+object GpuWindowExec extends Arm with Logging {
```

Comment:
nit: I don't think you need this logging here.

Reply:
thought I caught that one, will remove.